src/gui/editors/AutomationEditor.cpp

```diff
@@ -735,14 +771,13 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 				( mouseEvent->buttons() & Qt::LeftButton &&
 						m_editMode == ERASE ) )
 		{
-			// int resolution needed to improve the sensitivity of
-			// the erase manoeuvre with zoom levels < 100%
-			int zoom = m_zoomingXModel.value();
-			int resolution = 1 + zoom * zoom;
-			for( int i = -resolution; i < resolution; ++i )
+			// removing automation point
+			if( pos_ticks < 0 )
 			{
-				m_pattern->removeValue( MidiTime( pos_ticks + i ) );
+				pos_ticks = 0;
 			}
+			removePoints( m_drawLastTick, pos_ticks );
+			Engine::getSong()->setModified();
```

Comment:
It will mark the project as modified if no points are removed. It should be moved into `removePoints()`.

Reply:
:+1:

Reply:
I remove points without checking if there are any there. So if I add ```bool pointsRemoved = false;``` and lets the delete loop set it to true, it is still going to set the project to changed as soon as the mouse moves just a bit without really removing anything.

Reply:
The behaviour right now is, deleting points with right click sets the project to changed but deleting by dragging over the point doesn't.

Reply:
@PhysSong I don't know quite how to move ```Engine::getSong()->setModified();``` to `removePoints()`. This is already an improvement from before when setModified() wasn't called at all. I suggest merging this 'as is', or I can have another look at it if someone can point me in the right direction but for now I think it's low priority.

Reply:
`Engine::getSong()->setModified();` is an UX problem. So I think it's out of scope a little bit. Since the point of this PR is on GUI part, merging as-is looks fine. Dropping that will be fine, too.
I can create a PR which fixes the UX issue later. I think it will be the best way. :+1:

Reply:
All right. I'll merge this tomorrow.